### ISSUE TITLE
Unable to print ranked columns

### ISSUE THREAD

**Issue by jensfinnas** [onyxfish/agate#397]
With the example data set I get:

```
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "env/lib/python2.7/site-packages/agate/table.py", line 1106, in print_table
    print_table(self, max_rows, max_columns, output)
  File "env/lib/python2.7/site-packages/agate/preview.py", line 67, in print_table
    max_places = max_precision(c[:max_rows])
  File "env/lib/python2.7/site-packages/agate/utils.py", line 149, in max_precision
    places = value.normalize().as_tuple().exponent * -1
AttributeError: 'int' object has no attribute 'normalize'
```

...when I run:

```python
import agate
table = agate.Table.from_csv("example_data.csv")
table.compute([("my_rank", agate.Rank("age"))]).print_table(max_rows=5)
```

**Comment by onyxfish** [comment]
Ah ha! Bug! This is because of #381. Although until now I did not have a test case, so thank you for providing one. Going to close this ticket out but you can monitor the other one for a fix. Thanks for reporting it!

**Comment by anderser** [comment]
I am having the same issue with latest version 1.1.1. All the columns I have tried it on have the Number type. The code used is similar to that of @jensfinnas, but with another dataset. Was this fixed in #381 or is there something we could do to contribute?
